<?xml version="1.0"?>
<launch>

  <arg name="drone_name" default="espdrone_104"/>
  <arg name="waypoints_list" default="[[-1.5, -1.0, -0.2, 0, 8.0], [-1.5, 0.0, 0.0, 0, 5.0], [-1.5, 0.0, 0.0, 90, 5.0], [-1.5, 0.0, 0.0, -90, 5.0], [-1.5, 0.0, 0.0, 0, 5.0],[-1.5, 0.0, -1.20, 0, 2.0], [-1.5, 0.0, -1.50, 0, 2.0], [-1.5, 0.0, -1.70, 0, 2.0]]"/>
  <arg name="transition_duration" default="10.0"/>
  <arg name="goal_threshold" default="0.10"/>

  <node pkg="espdrone_utils" type="send_waypoints.py" name="waypoint_sender" output="screen">
    <param name="drone_name" type="string" value="$(arg drone_name)"/>
    <param name="waypoints_list" type="string" value="$(arg waypoints_list)"/>
    <param name="transition_duration" type="double" value="$(arg transition_duration)"/>
    <param name="goal_threshold" type="double" value="$(arg goal_threshold)"/>
  </node>

</launch>